### PR TITLE
[KOGITO-2997] BDD: Maven native build in container

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-2997

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by radtriste** [comment]
will test native part once release 0.14 is done to avoid to overload the cluster

File: Jenkinsfile.deploy [hunk 1]
```diff
@@ -168,7 +170,8 @@ pipeline {
                     
                     try {
                         // Optaplanner taking a lot of resources, we should not build in parallel
-                        sh "make build-examples-images tags='@native' concurrent=1 ${getBDDParameters('never', false)}"
+			// There seems to be a problem with podman executed from the BDD tests ... Using docker instead for now ...
```
Comment (radtriste) [inline Jenkinsfile.deploy:173]:
```suggestion
			  // There seems to be a problem with podman executed from the BDD tests ... Using docker instead for now ...
```

**Review by Sgitario — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]

**Comment by radtriste** [comment]
Seems like CI Openshift is having perf problems ... @msmagnanijr @Kaitou786 to you to have a look ...

I tested [on the side](https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/custom/job/tradisso/job/kogito-operator-deploy/16/console) the new `native` build and it works well. but we have to use `docker` as container engine for this particular case ...

@ricardozanini @spolti Feel free to merge once CI is corrected

**Pull request merged**